tests/notifier/test_keep_alive_monitor.py

```diff
@@ -68,20 +69,18 @@ def callback(events: List[Event]):
 
         begin_tp = datetime.now()
 
-        for _ in range(self.threshold_seconds):
-            self.keep_alive_monitor.process_events(self.keep_alive_events)
-            sleep(1)
+        with vcr.use_cassette("tests/cassette/keep_alive_monitor_remote_ping.yaml", record_mode="none") as cass:
+            for _ in range(self.threshold_seconds):
+                self.keep_alive_monitor.process_events(self.keep_alive_events)
+                sleep(1)
 
-        while not received_high_priority_event:
-            logging.info(f"Waiting for high priority event, this should only take {self.threshold_seconds}s")
-            sleep(1)
+            while not received_high_priority_event:
+                logging.info(f"Waiting for high priority event, this should only take {self.threshold_seconds}s")
+                sleep(1)
+            assert cass.play_count == 2, "Remote ping request did not happen as expected."
```

Comment:
I’d recommend `self.assertEqual` from `unittest` for consistency and I think logging is a bit nicer?